src/index.js

```diff
@@ -2,6 +2,8 @@ const React = require('react')
 const PropTypes = require('prop-types')
 const { IntlProvider, addLocaleData } = require('react-intl')
 const reduce = require('lodash/reduce')
+const keys = require('lodash/keys')
+const assign = require('lodash/assign')
```

Comment:
Using lodash here because we don't know if the environment has support for Object.assign and Object.keys